src/App.vue

```diff
@@ -822,35 +831,51 @@ export default class App extends Mixins(
     return userInfo
   }
 
+  /**
+   * Gets current account from object in session storage.
+   * Wait up to 10 sec for current account to be synced (typically by SbcHeader).
+   */
+  private async getCurrentAccount (): Promise<any> {
+    let account: any
+    for (let i = 0; i < 100; i++) {
+      const currentAccount = sessionStorage.getItem(SessionStorageKeys.CurrentAccount)
+      account = JSON.parse(currentAccount)
+      if (account) break
+      await sleep(100)
+    }
```

Comment:
I haven't seen this count get above 1 in this UI, but I poached this code from BC Registry UI where I sometimes saw delays of 2-3 seconds before CURRENT_ACCOUNT was available in session storage. So I think it's good to have this here.